person.rb

```diff
@@ -34,4 +36,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
 end
```

Comment:
- [ ] Great job so far in your implementation! 🚀 However, it seems that your `add_book_rental` method might not be working as expected. The `add_book_rental` method should take two parameters, `date` and `book`. Additionally, the person itself should also be passed as a parameter to create the new rental, achieving the 'has-many' association. 😊

To achieve this, consider implementing your method like this:

```ruby
  def add_book_rental(book, date)
     Rental.new(date, book, self)
  end
```

This revised approach will ensure that the rental is associated with the book correctly. 

Happy coding! 😊🚀👨‍💻